lib/ansible/runner/filter_plugins/core.py

```diff
@@ -270,6 +270,62 @@ def get_encrypted_password(password, hashtype='sha512', salt=None):
 def to_uuid(string):
     return str(uuid.uuid5(UUID_NAMESPACE_ANSIBLE, str(string)))
 
+def comment(text, style='plain', **kw):
+    # Predefined comment types
+    comment_styles = {
+        'plain': {
+            'decoration': '# '
+        },
+        'erlang': {
+            'decoration': '% '
+        },
+        'c': {
+            'decoration': '// '
+        },
+        'cblock': {
+            'beginning': '/*',
+            'decoration': ' * ',
+            'end': ' */'
+        },
+        'xml': {
+            'beginning': '<!--',
+            'decoration': ' - ',
+            'end': '-->'
+        }
+    }
+
+    # Pointer to the right comment type
+    style_params = comment_styles[style]
+
+    # Default params
+    p = {
+        'newline': '\n',
+        'beginning': '',
+        'prefix': (
+          kw['decoration'] if 'decoration' in kw else style_params['decoration']
+        ).rstrip(),
+        'prefix_count': 1,
+        'decoration': '',
+        'postfix': (
+          kw['decoration'] if 'decoration' in kw else style_params['decoration']
+        ).rstrip(),
+        'postfix_count': 1,
+        'end': ''
+    }
+
+    # Update default params
+    p.update(style_params)
+    p.update(kw)
+
+    return "%s%s%s%s%s%s" % (
+        "%s%s" % (p['beginning'], p['newline']) if p['beginning'] else '',
+        str("%s%s" % (p['prefix'], p['newline'])) * int(p['prefix_count']),
+        p['decoration'],
+        text.replace(p['newline'], "%s%s" % (p['newline'], p['decoration'])),
+        p['newline'].join([''] + [p['postfix'] for x in range(p['postfix_count'])]),
+        "%s%s" % (p['newline'], p['end']) if p['end'] else ''
+    )
+
```

Comment:
It would be great to expand this out so that it's several if-else blocks instead of a python one-liner.  It's kinda neat to be able to write these but for someone to read and have to modify it later can be very, very intimidating.  We're writing python and not perl after all ;-)